### ISSUE TITLE
TabListWrapper does not have the tab list role

### ISSUE THREAD

**Issue by carolyncole** [pulibrary/lux-design-system#673]
### Steps to recreate
We get an error in our rspec axe test
<img width="1762" height="692" alt="Image" src="https://github.com/user-attachments/assets/bb17b383-b517-4794-866c-eb5fcd6ac02d" />

### URL
https://dequeuniversity.com/rules/axe/4.13/aria-required-parent?application=axeAPI

### Severity

Critical

### WCAG Conformance level (A, AA, AAA)
A

### Recommended fix

Add the role "tablist" to the UL

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ul has a role "tablist"
